webAO/client.js

```diff
@@ -1016,15 +1016,15 @@ class Client extends EventEmitter {
 		for (let i = 1; i <= ini.emotions.number; i++) {
 			const emoteinfo = ini.emotions[i].split("#");
 			const esfx = ini.soundn[i] ? ini.soundn[i] : "0";
```

Comment:
missed one?

Reply:
soundN is the sfx name, 0 is used to indicate that there is no sfx to be played

Reply:
le sigh